Removed youtube-dl refs and links from CONTRIBUTING.md (#987)
yt-dlp/yt-dlp: CONTRIBUTING.md

For bug reports, this means that your report should contain the *complete* output of yt-dlp when called with the `-v` flag. The error message you get for (most) bugs even says so, but you would not believe how many of our bug reports do not contain this information.

If your server has multiple IPs or you suspect censorship, adding `--call-home` may be a good idea to get more diagnostics. If the error is `ERROR: Unable to extract ...` and you cannot reproduce it from multiple countries, add `--dump-pages` (warning: this will yield a rather large output, redirect it to the file `log.txt` by adding `>log.txt 2>&1` to your command-line) or upload the `.dump` files you get when you add `--write-pages` [somewhere](https://gist.github.com/).

**Site support requests must contain an example URL**. An example URL is a URL you might want to download, like `https://www.youtube.com/watch?v=BaW_jenozKc`. There should be an obvious video present. Except under very special circumstances, the main page of a video service (e.g. `https://www.youtube.com/`) is *not* an example URL.

###  Are you using the latest version?

> Before reporting any issue, type `yt-dlp -U`. This should report that you're up-to-date. About 20% of the reports we receive are already fixed, but people are using outdated versions. This goes for feature requests as well.

Reviewer: IMO this 20% thingy doesn't apply anymore so better to remove it altogether.
```suggestion
Before reporting any issue, type `yt-dlp -U`. This should report that you're up-to-date. This goes for feature requests as well.
```

###  Is the issue already documented?

> Make sure that someone has not already opened the issue you're trying to open. Search at the top of the window or browse the [GitHub Issues](https://github.com/yt-dlp/yt-dlp/search?type=Issues) of this repository. If there is an issue, feel free to write something along the lines of "This affects me as well, with version 2015.01.01. Here is some more information on the issue: ...". While some issues may be old, a new post into them often spurs rapid activity.

Reviewer: I know this is an example but maybe update the version just this time.
```suggestion
Make sure that someone has not already opened the issue you're trying to open. Search at the top of the window or browse the [GitHub Issues](https://github.com/yt-dlp/yt-dlp/search?type=Issues) of this repository. If there is an issue, feel free to write something along the lines of "This affects me as well, with version 2021.09.02. Here is some more information on the issue: ...". While some issues may be old, a new post into them often spurs rapid activity.
```

###  Is anyone going to need the feature?

Only post features that you (or an incapacitated friend you can personally talk to) require. Do not post features because they seem like a good idea. If they are really useful, they will be requested by someone who requires them.

###  Is your question about yt-dlp?

It may sound strange, but some bug reports we receive are completely unrelated to yt-dlp and relate to a different, or even the reporter's own, application. Please make sure that you are actually using yt-dlp. If you are using a UI for yt-dlp, report the bug to the maintainer of the actual application providing the UI. On the other hand, if your UI for yt-dlp fails in some way you believe is related to yt-dlp, by all means, go ahead and report the bug.

# DEVELOPER INSTRUCTIONS

> Most users do not need to build yt-dlp and can download the builds or get them from their distribution.

Reviewer: You can link the GH releases page here

### Adding support for a new site

> If you want to add support for a new site, first of all **make sure** this site is **not dedicated to copyright infringement**. yt-dlp does **not support** such sites thus pull requests adding support for them **will be rejected**.

Reviewer: It wouldn't be a bad idea to port this `can-you-add-support-for-this-anime-video-site` aswell

                # TODO more properties (see yt_dlp/extractor/common.py)
            }
    ```
5. Add an import in [`yt_dlp/extractor/extractors.py`](https://github.com/yt-dlp/yt-dlp/blob/master/yt_dlp/extractor/extractors.py).
6. Run `python test/test_download.py TestDownload.test_YourExtractor`. This *should fail* at first, but you can continually re-run it until you're done. If you decide to add more than one test, then rename ``_TEST`` to ``_TESTS`` and make it into a list of dictionaries. The tests will then be named `TestDownload.test_YourExtractor`, `TestDownload.test_YourExtractor_1`, `TestDownload.test_YourExtractor_2`, etc. Note that tests with `only_matching` key in test's dict are not counted in.
7. Have a look at [`yt_dlp/extractor/common.py`](https://github.com/yt-dlp/yt-dlp/blob/master/yt_dlp/extractor/common.py) for possible helper methods and a [detailed description of what your extractor should and may return](https://github.com/yt-dlp/yt-dlp/blob/0001fcb586c3ab297cd48c77ddd6f5d40546dac4/yt_dlp/extractor/common.py#L105-L366). Add tests and code for as many as you want.

> 8. Make sure your code follows [yt-dlp coding conventions](#yt-dlp-coding-conventions) and check the code with [flake8](https://flake8.pycqa.org/en/latest/index.html#quickstart):

Reviewer: we donot have `#yt-dlp-coding-conventions`
Maybe u can port them aswell

        $ flake8 yt_dlp/extractor/yourextractor.py

> 9. Make sure your code works under all [Python](https://www.python.org/) versions claimed supported by yt-dlp, namely 2.6, 2.7, and 3.2+.

Reviewer: ```suggestion
9. Make sure your code works under all [Python](https://www.python.org/) versions claimed supported by yt-dlp, namely 3.6+
```
Tho the whole line should be reworded imo
